ligo/helpers.mligo

```diff
@@ -8,11 +8,69 @@
 #include "errors.mligo"
 #include "math.mligo"
 
-let sqrt_price_move (liquidity : nat) (sqrt_price : x80n) (dx : nat) : x80n =
+(*  Calculate the new `sqrt_price` after a deposit of `dx` `x` tokens.
+    Derived from equation 6.15:
+        Δ(1 / √P) = Δx / L
```

Comment:
In the article, without the parentheses, this was really a bit confusing, good to see them here.